src/proptests/raw.rs

```diff
@@ -21,7 +21,11 @@ proptest! {
         let out_value = block_on(
             client.get(pair.key().clone())
         ).unwrap();
-        assert_eq!(Some(Value::from(pair.value().clone())), out_value);
+
+        match out_value {
+            None =>assert!(pair.value().is_empty()),
```

Comment:
It seems that the project is not fully reformatted?

Reply:
This piece of code inside a macro is not automatically formatted by rustfmt.